src/components/ShowingXofX/ShowingXofX.tsx

```diff
@@ -10,8 +10,8 @@ interface ShowingXofXProps {
 const ShowingXofX = ({ from, to, total, hideTotal }: ShowingXofXProps) => {
     return (
         <Typography sx={{ p: 0, mb: 1 }}>
-            Showing {from || 0}-{to || 0}{" "}
-            {!hideTotal && <>of {total || 0} results</>}
+            Showing {from || 0}-{to || 0}
```

Comment:
Quick fix to add a fullstop that makes it more readable when the ">100 results" message displays.